src/Cli/dotnet/commands/dotnet-workload/install/WorkloadManifestUpdater.cs

```diff
@@ -419,7 +470,41 @@ private async Task<bool> NewerManifestPackageExists(ManifestId manifest)
             }
         }
 
-        private IEnumerable<(ManifestId Id, ManifestVersionWithBand ManifestWithBand)> ParseRollbackDefinitionFile(string rollbackDefinitionFilePath)
+        public IEnumerable<ManifestVersionUpdate> ParseRollbackDefinitionFiles(IEnumerable<string> rollbackFilePaths)
```

Comment:
`SdkDirectoryWorkloadManifestProvider.GetAvailableWorkloadSets` has logic that does the same thing as this method.  Could this be factored so there's only one implementation?

Reply:
They don't look like they do the same thing. That method finds all installed workload sets and returns the ones that match this feature band. This takes in paths to workload sets and parses them into a set of manifest version updates. It's basically "what happens after you call SdkDirectoryWorkloadManifestProvider.GetAvailableWorkloadSets"

Reply:
The methods don't do the same thing, but inside the loop in `GetAvailableWorkloadSets` it seemed like it was doing the same thing (aggregating multiple json files into a single workload set).